src/components/scenes/PluginViewListScene.js

```diff
@@ -189,22 +210,15 @@ class PluginList extends Component<Props> {
     const countryData = COUNTRY_CODES.find(country => country['alpha-2'] === countryCode)
 
     // Pick a filter based on our direction:
-    const filter: BuySellFilter = direction === 'buy' ? buyPluginFilter : direction === 'sell' ? sellPluginFilter : buySellPluginFilter
-
-    // Remove plugins that don't exist in the filter:
-    let plugins = allPlugins.filter((plugin: BuySellPlugin) => filter[plugin.name.toLowerCase()] != null)
-
-    // Remove plugins that don't match our country:
-    if (countryData != null) {
-      plugins = plugins.filter((plugin: BuySellPlugin) => plugin.pluginId === 'custom' || filter[plugin.name.toLowerCase()].countryCodes[countryCode])
-    }
+    let plugins =
+      direction === 'buy'
+        ? getBuyPlugins(Platform.OS, countryCode)
+        : direction === 'sell'
+          ? getSellPlugins(Platform.OS, countryCode)
+          : getAllPlugins(Platform.OS, countryCode)
```

Comment:
From here on out we will only support showing either buy or sell plug-ins but not all. Please remove the all case. There is only one part of the app that calls and with all and that can be changed to request the buy list